File: java/src/model/Arquero.java
package model;

// Herencia
public class Arquero extends Personaje {

    // Constructor
    public Arquero(int id, String nombre, int nivel) {

        // constructor padre
        super(id, nombre, nivel);
    }

    // Sobrescritura
    @Override
    public void habilidadEspecial() {

        System.out.println("Disparo de flecha");
    }
}

File: python/model/arquero.py
from model.personaje import Personaje

# Herencia
class Arquero(Personaje):

    # Constructor
    def __init__(self, id, nombre, nivel):

        # constructor padre
        super().__init__(id, nombre, nivel)

    # sobrescritura
    def habilidad_especial(self):

        print("Disparo de flecha")